runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -3136,6 +3136,15 @@ void jitMethodBreakpointed(J9VMThread * vmThread, J9Method *j9method)
  */
 void jitIllegalFinalFieldModification(J9VMThread *currentThread, J9Class *fieldClass)
    {
+#if defined(J9VM_OPT_JITSERVER)
```

Comment:
compInfo is not defined at this point

Reply:
We should have got the jitConfig like below:
`J9JITConfig * jitConfig = currentThread->javaVM->jitConfig;`
Maybe it compiles because there is a global somewhere.

Reply:
Good eye! I can't believe that compiled.